rezervo/api/user.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import APIRouter, Depends, HTTPException
+from pydantic import BaseModel
+from sqlalchemy.orm import Session
+from starlette import status
+
+from rezervo.api.common import get_db, token_auth_scheme
+from rezervo.auth import auth0
+from rezervo.database import crud
+from rezervo.settings import Settings, get_settings
+
+router = APIRouter()
+
+
+class User(BaseModel):
+    name: str
+
+
+@router.put("/user", response_model=User)
+def upsert_user(
+    user: User,
```

Comment:
This seems useful, but we should retrieve the username from Auth0 directly, instead of allowing any username from the client.

Below is an existing example of retrieving the username from `jwt_sub`

https://github.com/mathiazom/rezervo/blob/84ba48946e52fbfe04b1ad05bf35d3ae3976776c/rezervo/api/sessions.py#L40

Reply:
Hey, that is cool. Fixed now ✅ https://github.com/mathiazom/rezervo/pull/41/commits/a806968e11f8a4dca73b1632bc65173b1968113b